csource/common.go

```diff
@@ -203,6 +225,8 @@ static void execute_command(const char* format, ...)
 
 int tunfd = -1;
 
+#define TUN_MTU 1500
```

Comment:
What will happen if this is less than packet size? Do we get only start of the packet and rest is discarded? Or the rest of the packet will remain in the kernel buffer and will break parsing of subsequent packets?
Either way 1500 does not look like the right number. If tail of a packet is discarded, then using MTU is confusing (because it is not related to MTU) and we need something along the lines:
```
// We need this to be only large enough to fit all of the header fields that we want to parse out.
#define SYZ_TUN_HEADER_SIZE 1000
```

Or, if the trailing is left in the kernel buffer, then I guess syzkaller can actually alter mtu value, and then we need something like:
```
// Maximum packet size (fuzzer can alter it from the default value).
#define SYZ_TUN_HEADER_SIZE (64<<10)
```
(is there an upper bound on MTU?)